documentation/modules/exploit/windows/fileformat/office_excel_slk.md

```diff
@@ -0,0 +1,50 @@
+## Introduction
+
+The .slk file format used by Microsoft Excel has the ability to execute local commands via the `EEXEC(cmd)` function. 
+This module takes advantage of this 'feature' to run a download-and-execute powershell command in order to spawn a session
+on the target.
+
+## Vulnerable Application
+
+  Microsoft Excel (tested on Excel 2016)
+
+## Verification Steps
+
+1. Start `msfconsole`
+2. `use exploit/windows/fileformat/office_excel_slk`
+3. `set LHOST [IP]`
+4. `set SRVHOST [IP]`
+4. `run`
+5. Open generated file and press 'Enable Content' in Excel
+
+## Options
+
+  **FILENAME**
+
+  The name of the generated .slk file. Default is a randomly generated file name.
```

Comment:
is this true? it looks like the default is `msf.slk` https://github.com/rapid7/metasploit-framework/pull/11249/files#diff-74a43747f24aa9c7a78661111c588930R53 (which would be easy to fingerprint, random name may be a better default)

Reply:
good point. I'll change in a second